pnpm-lock.yaml

```diff
@@ -1,9 +1,5 @@
 lockfileVersion: '6.0'
 
-settings:
-  autoInstallPeers: true
-  excludeLinksFromLockfile: false
```

Comment:
Any reason why these settings have been removed?

Reply:
this is not removed instead it is shifted to last of the file, this happened on running `pnpm install`